app.rb

```diff
@@ -148,6 +148,11 @@ def get_param(key, acceptable_values)
     end
   end
 
+  get '/failure_to_proof' do
+    puts 'Failure to Proof :('
+    erb :failure_to_proof#, layout: true
```

Comment:
Can we remove the commented code?

```suggestion
    erb :failure_to_proof
```